apps/kg-explorer/src/app/app.component.scss

```diff
@@ -1,8 +1,18 @@
 :host {
-  display: block;
+  display: grid;
```

Comment:
This grid setup seems unnecessary and might prevent proper scrolling of the header, etc. Instead set min/max height on the content (maybe with `height: 100%`). Experiment with flex properties inside the `.content` to get the right behavior